package.json

```
@@ -58,7 +58,7 @@
   "dependencies": {
     "basekick": "^3.0.0",
     "classnames": "^2.2.5",
-    "lodash": "^4.17.4",
+    "lodash": "^4.17.11",
```

```suggestion
    "lodash": "^4.17.14",
```

That said, is this PR necessary? Consumers can independently pin a version >4.17.11 due to the caret range.

---

I figured this would fix it once rather than every consumer (who potentially isn't using lodash in their own project) having to add it to their own package.json file.

Also, if that is the recommended solution does this become a "peer dependency"?